pkg/agent/endpoints/workload/handler.go

```diff
@@ -180,35 +207,68 @@ func (h *Handler) FetchX509SVID(_ *workload.X509SVIDRequest, stream workload.Spi
 	for {
 		select {
 		case update := <-subscriber.Updates():
-			metrics.IncrCounter([]string{workloadApi, "update"}, 1)
-
 			start := time.Now()
-			err := h.sendX509SVIDResponse(update, stream)
+			err := h.sendX509SVIDResponse(update, stream, metrics, selectors)
 			if err != nil {
 				return err
 			}
 
-			metrics.MeasureSince([]string{workloadApi, "update_latency"}, start)
+			metrics.MeasureSince([]string{workloadApi, "svid_response_latency"}, start)
```

Comment:
I'm not sure if this measurement is meaningful. Do we know what we'd do with this metric? It's measuring how long it took to send the update down the stream...

Reply:
Well, neither do I...I believe it could be used to trigger some kind of alarm if that metric gets to some determined value...or maybe it could be used to correlate how the number of connections affects the latency in serving SVIDs to workloads...